### PR TITLE
Use ETHAmount where possible

### PR DESCRIPTION
**Pull request by tomquirk**
## What does this PR do and why?

- Mostly everywhere where we use `currency="ETH"`, use `<ETHAmount />` component instead.
- A few smol improvements to ETHAmount internals

## Screenshots or screen recordings

---

## Acceptance checklist

- [x] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [x] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [x] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

File: src/components/Project/SpendingStats.tsx [hunk 1]
```diff
@@ -84,16 +84,18 @@ export default function SpendingStats({
...
-            <Space>
-              <Balance address={ownerAddress} />
-              <Trans>
-                <CrownFilled /> owner balance
-              </Trans>
-            </Space>
-          </Tooltip>
+          <Space>
+            <Balance address={ownerAddress} />
+            <Tooltip
```
Comment (tomquirk) [inline src/components/Project/SpendingStats.tsx:89]: unrelated fix. makes tooltip only apply to the `owner balance` text (`<Balance />` already has its own tooltip)

File: src/components/currency/ETHAmount.tsx [hunk 2]
```diff
@@ -7,48 +7,47 @@ import { betweenZeroAndOne } from 'utils/bigNumbers'
...
-const MAX_PRECISION = 6
+const MIN_AMOUNT = parseWad(0.00001)
 
 /**
  * Render a given amount formatted as ETH. Displays USD amount in a tooltip on hover.
  */
 export default function ETHAmount({
   amount,
-  precision = 0,
+  precision = PRECISION_ETH,
```
Comment (tomquirk) [inline src/components/currency/ETHAmount.tsx:19]: default to `4` instead

File: src/components/v2/V2Project/V2ManageTokensSection/V2RedeemModal.tsx [hunk 3]
```diff
@@ -177,7 +177,7 @@ export default function V2RedeemModal({
               </span>
             </Trans>
           </p>
-          <p style={statsStyle}>
+          <div style={statsStyle}>
```
Comment (tomquirk) [inline src/components/v2/V2Project/V2ManageTokensSection/V2RedeemModal.tsx:180]: Unrelated console error fix: "Cant have `div` inside `p`

**Comment by tomquirk** [comment]
@wraeth-eth sneaky review pls?

File: src/components/Project/VolumeStatLine/VolumeStatLine.tsx [hunk 4]
```diff
@@ -59,7 +59,7 @@ export const VolumeStatLine = ({
           >
             <ETHAmount
               amount={totalVolume ?? BigNumber.from(0)}
-              precision={4}
+              precision={2}
```
Comment (johnnyd-eth) [inline src/components/Project/VolumeStatLine/VolumeStatLine.tsx:62]: Back this. One smol step in descreaing number overload lol

File: src/components/currency/ETHAmount.tsx [hunk 5]
```diff
@@ -7,48 +7,47 @@ import { betweenZeroAndOne } from 'utils/bigNumbers'
 import CurrencySymbol from '../CurrencySymbol'
 
 import ETHToUSD from './ETHToUSD'
+import { PRECISION_ETH } from 'constants/currency'
 
-const MAX_PRECISION = 6
+const MIN_AMOUNT = parseWad(0.00001)
```
Comment (johnnyd-eth) [inline src/components/currency/ETHAmount.tsx:12]: **Clean**
